example/Bar13.purs

```diff
@@ -0,0 +1,185 @@
+module Bar13 where
+import Data.Array hiding (init)
+import Data.Maybe
+import Data.Tuple
+import Data.Tuple.Nested
+
+import ECharts.Chart
+import ECharts.Options.Unsafe
+import Data.Maybe
+import Utils
+
+options =
+  {
+    title:
+      {
+        text: "Another chinese stuff",
+        subtext: "Subtitle here"
+      },
+    tooltip:
+      {
+        trigger: "axis",
+        axisPointer:
+          {
+            "type": "cross",
+            lineStyle:
+              {
+                "type": "dashed",
+                width: 1
+              }
+          },
+        formatter: (\p ->
+                     p.seriesName <> " : [ " <>
+                     fromMaybe "" (p.value !! 0) <> ", " <>
+                     fromMaybe "" (p.value !! 1) <> "]")
+      },
+    legend:
+      {
+        "data": ["数据", "数据"]
```

Comment:
Can you please use English for all examples? It will make the examples much easier to read and understand for English speakers (and the rest of the library is in English, too, including all module and identifier names).